lib/hanami/assets/app_config.rb

```diff
@@ -4,21 +4,21 @@
 require_relative "config"
 
 module Hanami
+  # @api private
   module Assets
-    # @since 2.0.0
-    # @api public
+    # App config for assets.
+    #
+    # This is NOT RELEASED as of 2.0.0.
+    #
+    # @api private
```

Comment:
Leaving this note as well as `@api private` for any code inside hanami that's not officially part of 2.0.0 (e.g. views, assets, etc.)